src/components/value-list/value-list.tsx

```diff
@@ -101,6 +102,34 @@ export class ValueList<
    */
   @Prop() selectionFollowsFocus = false;
 
+  /**
+   * localize the screen reader text when drag button is focused
```

Comment:
@geospatialem Can you give the doc an editorial pass? Some things I notice:

1. Inconsistent capitalization
2. `place holder` -> `placeholder`
3. placeholder names should be camelCased
4. `indexes` -> `indices`? (I'll defer to you on this one as both are correct)